contrib/init/upstart/docker.conf

```diff
@@ -70,3 +70,25 @@ post-start script
 		echo "$DOCKER_SOCKET is up"
 	fi
 end script
+
+post-stop script
+        SHUTDOWN_RUNLEVELS='06'
+        if runlevel | cut -d' ' -f2 | grep -q [$SHUTDOWN_RUNLEVELS]; then
+                for container in $(docker-runc list -q); do
+                        docker-runc kill "$container" || true
```

Comment:
I would recommend the Docker daemon itself be modified to handle this more appropriately in some way, or that we figure out how to get the init system to do what's necessary for these containers -- how do other init systems handle these live-restore containers?